src/middlewares/snapshot-middleware.ts

```diff
@@ -0,0 +1,34 @@
+import { PolarisGraphQLContext } from '@enigmatis/polaris-common';
+
+export class SnapshotMiddleware {
+    public getMiddleware() {
+        return async (
+            resolve: any,
+            root: any,
+            args: any,
+            context: PolarisGraphQLContext,
+            info: any,
+        ) => {
+            let startIndex;
```

Comment:
i don't remember if in mgf it's like this but maybe add a logger.debug and write a relevant message("middleware start job).

Reply:
i don't think these logs are necessary, and potentially complicate the code
@chenshoo should i include logs?